### DISCUSSION TITLE
Source code examples for  Learning Modern Linux

### DISCUSSION THREAD

**Discussion by jacekkowalczyk82** [mhausenblas/modern-linux.info#1] [Q&A] [answered]
Hello, 
Today I started reading [Learning Modern Linux](https://learning.oreilly.com/library/view/learning-modern-linux/9781098108939/preface01.html#idm45805455313296) and I wanted to see checkout the code examples for the book. They should be here in this repo, but I did not found it here. 
Where can I found them? 
-----
BR 
Jacek Kowalczyk

**Comment by mhausenblas** [comment, marked as answer]
Thanks a lot for your interest in the book, appreciate the feedback and indeed it seems I missed pushing the code into this repo. I will do that today and sorry about this oversight (production-related, entirely on me).

**Comment by mhausenblas** [comment]
@jacekkowalczyk82 the code is now available at https://github.com/mhausenblas/modern-linux.info/tree/main/code
